api_app/analyzers_manager/file_analyzers/cape_sandbox.py

```diff
@@ -0,0 +1,69 @@
+import time
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class CAPEsandbox(FileAnalyzer):
+    def set_params(self, params):
+        self.__token = self._secrets["token"]
+        self.__vm_name = params.get("VM_NAME", "win7x64_8")
+
+    def run(self):
+        API_URL: str = "https://www.capesandbox.com/apiv2/tasks/create/file/"
+        headers = {
+            "Authorization": f"Token {self.__token}",
+        }
+
+        files = {
+            "file": (self.filename, self.read_file_bytes()),
+        }
+
+        data = {"machine": self.__vm_name}
+
+        response = requests.post(API_URL, headers=headers, files=files, data=data)
+        response_json = response.json()
+        if not response_json["error"]:
+            to_respond = {}
+            to_respond["result_url"] = response_json["url"]
+            task_id = response_json["data"]["task_ids"][0]
+            status_api = "https://www.capesandbox.com/apiv2/tasks/status/" + str(
+                task_id
+            )
+            while True:
```

Comment:
we should control the polling. There are a lot of other analyzers who perform this kind of polling. Please go look into them. 

plus, `while True` should never be used because a simple error could bring infinite loops

Reply:
Won't blame Aditya, since this is something I recommended in the other PR. https://github.com/intelowlproject/IntelOwl/pull/792#discussion_r779122902

my idea was that we can keep polling until celery's SoftTimeLimit exception is raised.

Reply:
makes sense but I think it is generally bad practice because we are assuming that another implicit mechanism would solve the eventual problem. I would say better safe and put a maximum number of possible loops

Reply:
Understood. Is there a particular analyzer you can redirect me to which does polling?

Reply:
Found it! https://github.com/intelowlproject/IntelOwl/blob/5998d1cbc0fec5ed4c72cc2fa3e052b0e0c05261/api_app/analyzers_manager/observable_analyzers/pulsedive.py#L77